adminStuff/addNewPost.php

```diff
@@ -0,0 +1,35 @@
+<?php
+include('config/init.php');
+if(isset($_REQUEST['submitItem'])){
+    $errors = array();
+
+    if(!@$_REQUEST['blogTitle']){
+        $errors['blogTitle'] = "*Required*";
+    }
+
+    if(!@$_REQUEST['body']){
+        $errors['body'] = "*Required*";
+    }
+    if(!@$_REQUEST['tagName']){
+        $errors['tagName'] = "*Required*";
+    }
+    if(sizeof($errors) == 0){
+        echo addPostQuery($_REQUEST['blogTitle'], $_REQUEST['body'], $_REQUEST['tagName']);
```

Comment:
This is the right idea, but I'd make sure you always do a redirect after inserting something rather than just echoing the results (this gets back to the double-submit problem).

You can even redirect back to this page with a URL parameter saying `completed=true` or something like that to tell yourself to display the confirmation.